airflow/providers/amazon/aws/operators/redshift_cluster.py

```diff
@@ -436,64 +438,55 @@ def __init__(
         aws_conn_id: str = "aws_default",
         deferrable: bool = False,
         poll_interval: int = 10,
+        max_attempts: int = 10,
```

Comment:
Can you update the doc string to describe this param. (and on the other hand you added `deferrable` to the docs but it's not in the signature here?

Reply:
Good catch.

>and on the other hand you added deferrable to the docs but it's not in the signature here

Not sure what you mean by this. Line 439 is the `deferrable` parameter.

Reply:
> Not sure what you mean by this. Line 439 is the deferrable parameter.
Oops ya, that was hidden by the code collapsing!